db/migrate/20141205103230_remove_expectations.rb

```diff
@@ -0,0 +1,13 @@
+class RemoveExpectations < Mongoid::Migration
+  def self.up
+    Edition.where(:expectation_ids.exists => true).each { |e| e.unset(:expectation_ids) }
+    Expectation.collection.drop
+  end
+
+  def self.down
+  end
+end
+
+class Expectation
+  include Mongoid::Document
+end
```

Comment:
Would be good to avoid this class if possible. There's a risk that running future migrations will re-create the collection because of it.  There should be a way to drop the collection without needing a model that uses it.